GUIs/ScriptRunnerPyQt5_GUI/scripFunc/scripUNIMULTI.py

```diff
@@ -518,46 +520,54 @@ def basConf(self, rdp):
-                    copy_tree('./configurations/winCONF/win10StigsRDPy/Group_Policy_Files/Machine',
+                    copy_tree('../configurations/winCONF/win10StigsRDPy/Group_Policy_Files/Machine',
                               r'c:\Windows\System32\GroupPolicy\Machine')
-                    copy_tree('./configurations/winCONF/win10StigsRDPy/Group_Policy_Files/User',
+                    copy_tree('../configurations/winCONF/win10StigsRDPy/Group_Policy_Files/User',
                               r'c:\Windows\System32\GroupPolicy\User')
-                    shutil.copy('./configurations/winCONF/win10StigsRDPy/Group_Policy_Files/gpt.ini',
+                    shutil.copy('../configurations/winCONF/win10StigsRDPy/Group_Policy_Files/gpt'
+                                '.ini',
                                 r'c:\Windows\System32\GroupPolicy\gpt.ini')
                     print('Successfully copied Group Policy Settings')
                 except IOError as e:
                     print("Unable to copy file. %s" % e)
                 path = 'C:/win10secRDPallowed.inf'
             elif rdp == 'no':
                 try:
-                    shutil.copy('./configurations/winCONF/win10StigsRDPn/Windows10Template11_17.inf',
-                                'C:/Windows10Template11_17.inf')
-                    copy_tree('./configurations/winCONF/win10StigsRDPn/gpoRDPn/Machine',
+                    shutil.copy('../configurations/winCONF/win10StigsRDPn/Windows10Template11_17'
+                                '.inf',
+                                r'c:\Windows10Template11_17.inf')
+                    copy_tree('../configurations/winCONF/win10StigsRDPn/gpoRDPn/Machine',
                               r'c:\Windows\System32\GroupPolicy\Machine')
-                    copy_tree('./configurations/winCONF/win10StigsRDPn/gpoRDPn/User',
+                    copy_tree('../configurations/winCONF/win10StigsRDPn/gpoRDPn/User',
                               r'c:\Windows\System32\GroupPolicy\User')
-                    shutil.copy('./configurations/winCONF/win10StigsRDPn/gpoRDPn/gpt.ini',
+                    shutil.copy('../configurations/winCONF/win10StigsRDPn/gpoRDPn/gpt.ini',
                                 r'c:\Windows\System32\GroupPolicy\gpt.ini')
                     print('Successfully copied Group Policy Settings')
                 except IOError as e:
                     print('Unable to copy file. %s' % e)
                 path = 'C:/Windows10Template11_17.inf'
             else:
                 raise ValueError('rdp should be either yes or no!')
-            command = r"secedit /configure /db C:\\windows\\security\\local.sdb /cfg {0}".format(path)
-            sub.call(command.split())
+            command = r"secedit /configure /db C:\\windows\\security\\local.sdb /cfg {0}".format(
+                path)
+            sub.Popen(command.split())
             command = 'gpupdate'
-            sub.call(command)
+            sub.Popen(command)
```

Comment:
Codacy found an issue: [subprocess call - check for execution of untrusted input.](https://app.codacy.com/manual/vipersniper0501/CP_Scripts2/pullRequest?prid=5927419)